tox.ini

```diff
@@ -54,6 +56,13 @@ deps = pip-tools
 commands = pip-compile --output-file requirements.txt requirements.in
 changedir = {toxinidir}
 
+[testenv:pysnooper]
+description = Ensure pip installs pysnooper
+deps = pip
```

Comment:
No need to install `pip`. Replace this by `pysnooper` if you want to install it off PyPI (that's probably _not_ what you want), or leave it empty.